x-pack/plugins/security_solution/cypress/tasks/host_risk.ts

```diff
@@ -0,0 +1,17 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
```

Comment:
Please wrap all the locators into a constant and extract them to our screens folder.

Reply:
Hi @MadameSheema, could you share a link to the documentation? I want to learn about this convention?